doc/_extensions/zephyr/domain/__init__.py

```diff
@@ -14,6 +14,7 @@
 - ``zephyr:code-sample::`` - Defines a code sample.
 - ``zephyr:code-sample-category::`` - Defines a category for grouping code samples.
 - ``zephyr:code-sample-listing::`` - Shows a listing of code samples found in a given category.
+- ``zephyr:board-catalog::`` - Show a listing of boards supported by Zephyr.
```

Comment:
Nit: Shows (same as previous line).

Reply:
Oops I think I forgot this one. Will keep it in mind for future updates. Thanks!